File: objectModel/Java/objectmodel/src/main/java/com/microsoft/commondatamodel/objectmodel/utilities/NameValuePair.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

package com.microsoft.commondatamodel.objectmodel.utilities;
/**
 * Name value pair type class
 */
public class NameValuePair {

  private String name;
  private String value;

  public String getValue() {
    return value;
  }

  public void setValue(final String value) {
    this.value = value;
  }

  public String getName() {
    return name;
  }

  public void setName(final String name) {
    this.name = name;
  }
}

File: objectModel/Python/cdm/utilities/namevaluepair.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from cdm.utilities import JObject

class NameValuePair(JObject):
    """
    Name value pair class
    """

    def __init__(self):
        super().__init__()

        self.name = None  # type: str
        self.value = None  # type: str
